blocks/library/embed/index.js

```diff
@@ -342,175 +342,175 @@ registerBlockType(
 	'core-embed/animoto',
 	getEmbedBlockSettings( {
 		title: 'Animoto',
-		icon: 'video-alt3',
+		icon: 'embed-video',
 	} )
 );
 registerBlockType(
 	'core-embed/cloudup',
 	getEmbedBlockSettings( {
 		title: 'Cloudup',
-		icon: 'cloud',
+		icon: 'embed-post',
```

Comment:
Cloudup - shouldn't it be `embed-photo` based on the list from #2120?

Reply:
Since you can put video and markdown documents there too, I went with a more generic "post" icon.